demo/Gemfile.lock

```diff
@@ -241,12 +262,12 @@ DEPENDENCIES
   jbuilder
   jsbundling-rails
   puma
-  rails (~> 7.0.0)
+  rails (~> 7.1.0)
   selenium-webdriver
   sprockets-rails
   sqlite3 (~> 1.4)
   tzinfo-data
   web-console
 
 BUNDLED WITH
-   2.4.13
+   2.4.1
```

Comment:
I think it is a good idea to keep up with newer versions of bundler using `bundle update --bundler`.  At least not downgrade bundler without a reason.

Reply:
Agreed. I was using an older Docker container. Sometimes it seems like a stable development platform can cause more problems than it solves. :frowning_face: